package.json

```
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "rm -rf dist && tsc",
     "prepublishOnly": "npm run generate && npm run build",
```

```suggestion
    "prepublishOnly": "npm run build && npm run generate",
```

@pavelkornev Shouldn't it first build (wipe dist, compile typescript) and then generate?

---

In this project it doesn't really matter as generate works via `ts-node-dev`, see line 10. The only reason we have generate here is to convert the entry point (index.ts) to JS file before publishing npm package.